### PR TITLE
Rename `ord wallet satoshis` to `ord wallet sats`

### PR DESCRIPTION
**Pull request by casey**

### CODE REVIEW

**Comment by rot13maxi** [comment]
yes please

**Review by rot13maxi — APPROVED** [review]
not that you need my approval. just putting another set of eyes on it. LGTM

**Comment by casey** [comment]
@rot13maxi LGTMs and "I want this"s are always welcome!

**Pull request merged**
